internal/data/kv_pair.go

```diff
@@ -1032,11 +1032,17 @@ func (rp kvPairRepo) CreateMetadataKvPairs(ctx context.Context, checkInfo biz.Ch
 				return err
 			}
 			if len(subKeys) > 0 {
-				if err := tx.Model(SubKeyInfo{}).WithContext(ctx).Clauses(clause.OnConflict{
-					Columns:   []clause.Column{{Name: "pub_key"}, {Name: "lock_hash"}},
-					UpdateAll: true,
-				}).Create(subKeys).Error; err != nil {
-					return err
+				for _, subKey := range subKeys {
+					var oldSubkey SubKeyInfo
+					err := tx.Model(SubKeyInfo{}).WithContext(ctx).Where("lock_hash = ? and pub_key = ? and credential_id = ?", subKey.LockHash, subKey.PubKey, subKey.CredentialId).First(&oldSubkey).Error
```

Comment:
maybe we can creat an index on the `lock_hash`

Reply:
The `lock_hash` currently has duplicate data

Reply:
Updated https://github.com/nervina-labs/cota-syncer/pull/35/commits/a59e763b5273d2ed869d42964b216c2d147ac1e2